stable/gitlab-runner/templates/deployment.yaml

```diff
@@ -0,0 +1,89 @@
+{{- if and (default "" .Values.gitlabUrl) (default "" .Values.runnerRegistrationToken) }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: {{ .Values.image }}
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+        command: ["/bin/bash", "/scripts/entrypoint"]
+        env:
+        - name: GITLAB_URL
+          value: {{ default "" .Values.gitlabUrl | quote }}
+        - name: REGISTRATION_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: {{ template "fullname" . }}
+              key: runner-registration-token
+        - name: KUBERNETES_IMAGE
+          value: {{ .Values.runners.image | quote }}
+        {{ if .Values.runners.privileged }}
+        - name: KUBERNETES_PRIVILEGED
+          value: true
+        {{ end }}
+        - name: KUBERNETES_NAMESPACE
+          value: {{ default "" .Values.runners.namespace | quote }}
```

Comment:
when not specified would it make sense to default to `Release.Namespace` instead?
ref: https://github.com/kubernetes/helm/blob/master/docs/charts.md#predefined-values